nextjs-starter/app/examples/get-recipe/Recipe.tsx

```diff
@@ -1,93 +1,81 @@
-"use client";
+'use client';
 
-import { Ingredient, PartIngredient, PartSteps, Recipe } from "@/baml_client";
-import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
-import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
-import { RecursivePartialNull } from "@/baml_client/async_client";
-import { Slider } from "@/components/ui/slider";
-import { ReactNode, useEffect, useState } from "react";
-import { CheckCircle, Loader2 } from "lucide-react";
-import { ScrollArea } from "@/components/ui/scroll-area";
+import type { Ingredient, PartIngredient, PartSteps, partial_types } from '@/baml_client';
+import { HookResultData, HookResultPartialData } from '@/baml_client/react/types';
```

Comment:
I'd like to bikeshed these names a bit (this might be the best), but it doesn't read at first glance

Reply:
Yeah, I agree. Open to ideas